README.md

```diff
@@ -2,18 +2,4 @@
 
 * Rubocop: opcion -R no existe. Por lo tanto ejecute bundle exec rubocop app spec -A para realizar la correcion. Spec no existe, por lo tanto he creado su carpeta para solucionar los errores.
 
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+* Card: (2) Crear modelo de User + implementar autenticación [![Build Status](https://travis-ci.com/wx-inc-trainings/me-rails.svg?branch=auth)](https://travis-ci.com/wx-inc-trainings/me-rails)
```

Comment:
change the branch name because when this change merge in master the auth branch doesn’t exist
```suggestion
* Card: (2) Crear modelo de User + implementar autenticación [![Build Status](https://travis-ci.com/wx-inc-trainings/me-rails.svg?branch=master)](https://travis-ci.com/wx-inc-trainings/me-rails)
```